src/main/groovy/org/broadinstitute/orsp/dataUseLetter/DataUseLetterFields.groovy

```diff
@@ -20,24 +20,13 @@ enum DataUseLetterFields {
     DISEASE_RESTRICTED_OPTIONS("diseaseRestrictedOptions"),
     PARASITIC_DISEASE("parasiticDisease"),
     CANCER("cancer"),
-    ENDOCRINE_DISEASE("endocrineDisease"),
-    ENDOCRINE_DIABETES("endocrineDiabetes"),
     MENTAL_DISORDER("mentalDisorder"),
     NERVOUS_DISEASE("nervousDisease"),
-    EYE_DISEASE("eyeDisease"),
-    EAR_DISEASE("earDisease"),
     RESPIRATORY_DISEASE("respiratoryDisease"),
     DIGESTIVE_DISEASE("digestiveDisease"),
-    INFLAMMATORY_DISEASE("inflammatoryDisease"),
-    SKIN_DISEASE("skinDisease"),
-    MUSCULO_SKELETAL_DISEASE("musculoskeletalDisease"),
-    GENITOURINARY_DISEASE("genitourinaryDisease"),
     CARDIOVASCULAR_DISEASE("cardiovascularDisease"),
-    PREGNANCY("pregnancy"),
-    CONGENITAL_MALFORMATION("congenitalMalformation"),
-    BLOOD_DISORDER("bloodDisorder"),
     OTHER_DISEASE("otherDisease"),
-    OTHER_DISEASE_SPECIFY("otherDiseaseSpecify"),
+    OTHER_DISEASE_DOID("otherDiseasesDOID"),
```

Comment:
Nit: naming. This naming suggests (not incorrectly) that the disease ID will be a DOID. This is a tight coupling to the ontology. While it is true that we are using DOID today, we may not be using DOID in the future. I think OTHER_DISEASE_ID is just as descriptive and yet does not preclude other ontology disease ids.